crates/types/src/net/mod.rs

```diff
@@ -246,6 +328,20 @@ macro_rules! define_message {
 //       @response_target = TargetName::AttachResponse,
 //   }
 // ```
+// Example (With Bilrost V2 protocol):
+// ```
+//   define_rpc! {
+//       @proto = ProtocolVersion::Bilrost,
+//       @request = AttachRequest (as AttachRequestBilrostType),
+//       @response = AttachResponse,
+//       @request_target = TargetName::ClusterController,
+//       @response_target = TargetName::AttachResponse,
+//   }
```

Comment:
This really needs to be rebased on my changes.

Reply:
Sure, once your changes are merged in I will do the necessary rework